src/components/CommentComposer/docs.t.js

```diff
@@ -0,0 +1,10 @@
+export default k => {
```

Comment:
Let's just follow the form component model and use component attributes like "placeholder" and "replyLabel", which will make the whole t() mocking obsolete.

Reply:
This doesn't scale well. If you compose a few of these components into larger ones, you'll end up having to pass dozens of props to the top-level component, and it increases the likelihood of a name conflict. Already now we have two components which would require a `replyLabel`: `CommentActions` and `CommentComposer`.

Reply:
Good point about composition, I agree. So let's stick to your current version and defer further improvements of the translations setup to later.